develop/parachain-devs/interoperability/transfer-assets-with-xcm.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Transfer Assets with XCM
+description: This tutorial will guide you through the process of transferring assets between parachains using Cross-Chain Message (XCM) instructions.
+---
+
+# Transfer Assets with XCM
+
+## Introduction
+
+In the tutorial on [Open Message Passing Channels](TODO:update-path){target=\_blank} you learned how to establish a two-way communication channel between chains by sending messages to the relay chain. You can apply a similar approach to send messages that enable a local chain to manage an account on a remote chain. This tutorial will teach you how to transfer assets between chains using Cross-Chain Message (XCM) instructions.
+
+To transfer assets between two chains, you have the options of [Teleporting Assets](https://wiki.polkadot.network/docs/learn-teleport){target=\_blank} or performing a [Reserve Asset Transfer](https://wiki.polkadot.network/docs/learn/xcm/journey/transfers-reserve){target=\_blank}. This tutorial will cover both methods.
+
+## Prerequisites
+
+Before you begin, verify the following:
+
+- You have a relay chain and a parachain running to test the XCM instructions
+- You have opened the message passing channel to allow communication between the two chains
+
+## Teleporting Assets
+
+Asset teleportation enables the transfer of digital assets, including fungible and non-fungible tokens, across different parachains. This process allows the transferred assets to function as if they were native to the receiving chain. To simulate the teleportation of assets, this tutorial showcases the transfer of fungible tokens between the Polkadot relay chain and the Asset Hub system parachain.
+
+To replicate the scenario, this guide uses [Chopsticks](https://github.com/AcalaNetwork/chopsticks){target=\_blank} to interact with the relay chain and the Asset Hub parachain. To setup the same environment, run the following command:
```

Comment:
I would just mention that this example teleports funds from Polkadot to Asset Hub. And mention that they can replicate it locally using Chopsticks by following the example in https://github.com/polkadot-developers/polkadot-docs/blob/master/develop/application-devs/tooling/chopsticks/overview.md#xcm-testing